adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -247,6 +252,15 @@ func (swagger *MgwSwagger) setXThrottlingTier() {
 	}
 }
 
+// resolve authHeader from swagger vendor extension map
+func resolveAuthHeader(vendorExtensions map[string]interface{}) string {
+	if xAuthHeader, ok := vendorExtensions[xAuthHeader]; ok {
```

Comment:
Does this works fro open API 3 as well as swagger 3

Reply:
Yes it is working for both